afc-submit-wizard.js

```diff
@@ -708,6 +866,23 @@ function prepareDraftText(page) {
 	// put AfC submission template
 	header += '{{subst:submit|1=' + (mw.util.getParamValue('username') || '{{subst:REVISIONUSER}}') + '}}\n';
 
+	// Check for best sources
+	sng = ui.sngInput.getValue().split(",");
+	sourcesExist = (sng[0] == "GNG") && (ui.bestsourcesInput1.getValue() || ui.bestsourcesInput2.getValue() || ui.bestsourcesInput3.getValue());
+	
+	if(sng[1] != "") {
+		sng = '[[WP:' + sng[0] + ']]#' + sng[1];
+	} else {
+		sng = '[[WP:' + sng[0] + ']]';
+	}
+	
+	header += '{{afc comment|1=';
+	header += 'I believe this article meets ' + sng + '. ';
```

Comment:
I think that's putting words in the user's mouth, since nowhere they say that explicitly.